db/process_message.c

```diff
@@ -75,11 +75,12 @@ extern int __berkdb_read_alarm_ms;
 #include "crc32c.h"
 #include "ssl_bend.h"
 #include "dohsql.h"
+#include "fdb_whitelist.h"
 
-#include <trigger.h>
-#include <sc_stripes.h>
-#include <sc_global.h>
-#include <logmsg.h>
+#include "trigger.h"
```

Comment:
Why do we replace brackets with quotes here? Are these files in db directory?

Reply:
They are local to the comdb2 dir. I believe good practice is to have only system includes in <>.

Reply:
Not really. The convention is to use quotes for headers in the same directory as the source file and use brackets for the rest.

Reply:
http://www.cplusplus.com/forum/general/24564/
<> tells compiler  to look for file in system dirs first.

Reply:
https://github.com/torvalds/linux/blob/master/drivers/android/binder.c#L54-L83